### PR TITLE
Arnej/use vespalib metrics in sentinel

### PR DESCRIPTION
**Pull request by arnej27959**
@havardpe please take a look
@hmusum FYI

### CODE REVIEW

**Comment by havardpe** [comment]
-- Found JNI: /usr/lib/jvm/java-openjdk/jre/lib/amd64/libjawt.so  
CMake Error at /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find BISON (missing: BISON_EXECUTABLE)
Call Stack (most recent call first):
  /usr/share/cmake3/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake3/Modules/FindBISON.cmake:226 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  document/src/vespa/document/select/CMakeLists.txt:3 (find_package)


-- Configuring incomplete, errors occurred!
See also "/source/CMakeFiles/CMakeOutput.log".

**Comment by vekterli** [comment]
Build failure is due to some missing package deps on docker due to new Flex/Bison parser. Should be fixed on HEAD.

File: configd/src/apps/sentinel/metrics.cpp [hunk 1]
```diff
@@ -4,18 +4,33 @@
...
 StartMetrics::StartMetrics()
-    : currentlyRunningServices(0), totalRestartsCounter(0), totalRestartsLastPeriod(1),
-      lastLoggedTime(0),
-      totalRestartsLastSnapshot(0),
-      snapshotStart(0),
-      snapshotEnd(0)
+    : metrics(SimpleMetricsManager::create(SimpleManagerConfig())),
+      producer(metrics),
+      currentlyRunningServices(0),
+      totalRestartsCounter(0),
```
Comment (havardpe) [inline configd/src/apps/sentinel/metrics.cpp]: how can the total restarts be 0 while the restarts in the last period is 1?

**Comment by havardpe** [comment]
looks ok, but having 4 different restart concepts with different meaning and type makes it kind of confusing.

**Pull request merged**